src/crypto-api.ts

```diff
@@ -298,6 +298,11 @@ export interface BootstrapCrossSigningOpts {
      * will not be uploaded to the server (which seems like a bad thing?).
      */
     authUploadDeviceSigningKeys?: UIAuthCallback<void>;
+
+    /**
+     * Called if the cross signing keys are imported from the secret storage
+     */
+    onCrossSigningKeysImport?: () => void;
```

Comment:
I'm not keen on exposing this to the application.

Maybe just pass an `onCrossSigningKeysImport` callback into the constructor of `CrossSigningIdentity` ?